### PR TITLE
Update Runtime Upgrade issue template

### PR DESCRIPTION
**Pull request by educlerici-zondax**
Update Runtime Upgrade issue template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/runtime-upgrade---upgrade-title-version-.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+labels: runtime-upgrade
+assignees: ''
+
+---
+
+# Paseo Testnet Runtime Upgrade Checklist
+
+## Preparation
+
+- [ ] **Upgrade Proposal Review**: Confirm that the upgrade proposal has been fully reviewed and approved.
```
Comment (carlosala) [inline .github/ISSUE_TEMPLATE/runtime-upgrade---upgrade-title-version-.md:14]: What do we exactly mean here? Is it the proposal of Polkadot? Some links might be worth

File: .github/ISSUE_TEMPLATE/runtime-upgrade---upgrade-title-version-.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+- [ ] **Schedule Upgrade Window**: Decide on and communicate the upgrade window to all stakeholders.
+- [ ] **Notify Stakeholders**: Distribute the communication plan through all relevant channels.
+- [ ] **Prepare Upgrade Guide**: Ensure detailed upgrade instructions are available and easy to follow.
+- [ ] **Validator Coordination**: Confirm that validators are informed and prepared for the upgrade.
+
+## Execution
+
+- [ ] **Final Pre-Upgrade Check**: Conduct a last-minute check to ensure all systems and stakeholders are ready.
+- [ ] **Deploy Upgrade**: Initiate the runtime upgrade process according to the planned procedure.
+- [ ] **Monitor Deployment**: Closely monitor the network for any immediate issues during the upgrade.
```
Comment (carlosala) [inline .github/ISSUE_TEMPLATE/runtime-upgrade---upgrade-title-version-.md:29]: Shall we comment here about timezones, preferred time to do it or something like that?

File: .github/ISSUE_TEMPLATE/runtime-upgrade---upgrade-title-version-.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+## Validation
+
+- [ ] **Verify Network Stability**: Check that the network returns to normal operation post-upgrade.
+- [ ] **Confirm Upgrade Success**: Validate that the new runtime version is correctly implemented across the network.
+- [ ] **Stakeholder Feedback**: Collect and review feedback from validators, developers, and users.
+
+## Post-Upgrade
+
+- [ ] **Update Documentation**: Revise all relevant documentation to reflect changes introduced by the upgrade.
```
Comment (carlosala) [inline .github/ISSUE_TEMPLATE/runtime-upgrade---upgrade-title-version-.md:39]: We should revise everything beforehand and then update accordingly in post-upgrade.

File: .github/ISSUE_TEMPLATE/runtime-upgrade---upgrade-title-version-.md (hunk 2 above)
Comment (educlerici-zondax) [reply .github/ISSUE_TEMPLATE/runtime-upgrade---upgrade-title-version-.md:29]: I like that, maybe use CET as a standard?

File: .github/ISSUE_TEMPLATE/runtime-upgrade---upgrade-title-version-.md (hunk 1 above)
Comment (educlerici-zondax) [reply .github/ISSUE_TEMPLATE/runtime-upgrade---upgrade-title-version-.md:14]: i tried to write if the core team is aligned with the upgrade coming

File: .github/ISSUE_TEMPLATE/runtime-upgrade---upgrade-title-version-.md (hunk 3 above)
Comment (educlerici-zondax) [reply .github/ISSUE_TEMPLATE/runtime-upgrade---upgrade-title-version-.md:39]: i agree with that, are you ok if i update the .md with that?

File: .github/ISSUE_TEMPLATE/runtime-upgrade---upgrade-title-version-.md (hunk 2 above)
Comment (carlosala) [reply .github/ISSUE_TEMPLATE/runtime-upgrade---upgrade-title-version-.md:29]: Yep, let's use CET/CEST 👍🏻

File: .github/ISSUE_TEMPLATE/runtime-upgrade---upgrade-title-version-.md (hunk 1 above)
Comment (carlosala) [reply .github/ISSUE_TEMPLATE/runtime-upgrade---upgrade-title-version-.md:14]: Let's make it more explicit 👍🏻

Comment (educlerici-zondax) [reply .github/ISSUE_TEMPLATE/runtime-upgrade---upgrade-title-version-.md]: already updated

File: .github/ISSUE_TEMPLATE/runtime-upgrade---upgrade-title-version-.md (hunk 2 above)
Comment (educlerici-zondax) [reply .github/ISSUE_TEMPLATE/runtime-upgrade---upgrade-title-version-.md]: already updated

File: .github/ISSUE_TEMPLATE/runtime-upgrade---upgrade-title-version-.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+about: Checklist for tracking the progress of a runtime upgrade on the Paseo Testnet.
+title: ''
+labels: runtime-upgrade
+assignees: ''
+
+---
+
+# Paseo Testnet Runtime Upgrade Checklist
+
+## Preparation Tasks
```
Comment (al3mart) [inline .github/ISSUE_TEMPLATE/runtime-upgrade---upgrade-title-version-.md:12]: Should we include an extra DB backup before enacting any upgrade ?

As an effort of having the means to a smooth recovery in the scenario where something fails for whatever reason. Avoiding network resets as much as we possibly can .

Comment (hbulgarini) [reply .github/ISSUE_TEMPLATE/runtime-upgrade---upgrade-title-version-.md:12]: i cannot agree more.

File: .github/ISSUE_TEMPLATE/runtime-upgrade---upgrade-title-version-.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+- [ ] **Chopsticks testing:** Fork Paseo relay-chain and its system chains and commit the targeted runtime upgrade.
+  - [ ] **Epoch and Era changes:** Make chopticks to produce the necessary amount of changes to change the Era and assure blocks are still produced.
+  - [ ] **XCM transfers:** Make a teleport to the Paseo assethub and PAS reserved transfer to any available parachain.
+- [ ] **Try-Runtime**: Run try-runtime and mke sure that no state inconsistences are reported.
+- [ ] **Polkadot Fellowship contact:** In case of any issues detected during testing, contact any Polkadot Fellowship member to address the concern. If no issues this can just be checked.
+- [ ] **Runtime upgrade on test instance:** Commit the Runtime upgrade on the Paseo test instance and leave it running for a whole day.
+
+## Execution
+
+- [ ] **DB Backup**: Ask any infrastructure provider to produce a backup of the network before commiting the upgrade.
```
Comment (hbulgarini) [inline .github/ISSUE_TEMPLATE/runtime-upgrade---upgrade-title-version-.md:27]: @al3mart i defined as part of the execution plan.

**Review by hbulgarini — APPROVED** [review]

**Review by al3mart — APPROVED** [review]

**Pull request merged**